pkg/csi/service/common/authmanager.go

```diff
@@ -284,7 +310,7 @@ func IsFileServiceEnabled(ctx context.Context, datastoreUrls []string, vc *vsphe
 			dsToFSEnabledMapToReturn[datastoreURL] = false
 		}
 	}
-	return dsToFSEnabledMapToReturn, nil
+	return dsToFSEnabledMapToReturn, clusterToFileServiceEnabledDsMap, nil
```

Comment:
clusterToFileServiceEnabledDsMap naming is not correct. It is cluster which is FS enabled and not datastore. So 
`FsEnabledClusterToDsMap` sounds good.

here and every where.

Reply:
Makes sense. Will change it at all places.

Reply:
Done.